src/main/java/com/example/provider/json/VehicleSerializer.java

```diff
@@ -57,6 +74,9 @@ public JsonElement serialize(Vehicle src, Type typeOfSrc, JsonSerializationConte
             .setBackToBackEnabled(src.getBackToBackEnabled())
             .setSupportedTripTypes(src.getSupportedTripTypesList())
             .setMaximumCapacity(src.getMaximumCapacity())
+            .setLastLocation(serializedLastLocation)
```

Comment:
nit: before we submit, let's just confirm we do manual testing on the different code paths added. example:

1. src.getLastLocation() is null and not null
2. src.getLastLocation() has heading vs has no heading

Reply:
@lucys-s were you able to test these cases?